<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxTitle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelTitle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Harman T500 Function Test System&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelVersion">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 85, 255);</string>
     </property>
     <property name="text">
      <string>Version: 1.0.0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxLogo">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelLogo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/images/images/MINAMI_logo.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelDate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 94, 173);</string>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSelector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListView" name="listViewSelector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>401</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(85, 85, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxStackView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>840</width>
      <height>420</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>810</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
